pkgs/development/compilers/gcc/8/default.nix

```diff
@@ -223,56 +220,19 @@ stdenv.mkDerivation ({
     CPATH =  makeSearchPathOutput "dev" "include" (optional (zlib != null) zlib);
     LIBRARY_PATH = makeLibraryPath (optional (zlib != null) zlib);
   } // optionalAttrs (targetPlatform != hostPlatform && libcCross != null) {
-    EXTRA_TARGET_FLAGS =
-      "-idirafter ${getDev libcCross}${libcCross.incdir or "/include"}"
-      + optionalString (!crossStageStatic)
-        " -B${libcCross.out}${libcCross.libdir or "/lib"}";
-    EXTRA_TARGET_LDFLAGS =
-      "-Wl,-L${libcCross.out}${libcCross.libdir or "/lib"}"
-      + (if crossStageStatic then
-          "-B${libcCross.out}${libcCross.libdir or "/lib"}"
-        else
-          "-Wl,-rpath,${libcCross.out}${libcCross.libdir or "/lib"}" +
-          " -Wl,-rpath-link,${libcCross.out}${libcCross.libdir or "/lib"}"
-      );
-=======
-  installTargets =
-    if stripped
-    then "install-strip"
-    else "install";
-
-  # https://gcc.gnu.org/install/specific.html#x86-64-x-solaris210
-  ${if hostPlatform.system == "x86_64-solaris" then "CC" else null} = "gcc -m64";
-
-  # Setting $CPATH and $LIBRARY_PATH to make sure both `gcc' and `xgcc' find the
-  # library headers and binaries, regarless of the language being compiled.
-  #
-  # Likewise, the LTO code doesn't find zlib.
-  #
-  # Cross-compiling, we need gcc not to read ./specs in order to build the g++
-  # compiler (after the specs for the cross-gcc are created). Having
-  # LIBRARY_PATH= makes gcc read the specs from ., and the build breaks.
```

Comment:
I think this stuff was deleted by mistake? I'll take a look

Reply:
should be in the `env` attrs

Reply:
Oh, OK.